src/freertos/platforms/posix/include/freertos_platform_config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __FREERTOS_PLATFORM_CONFIG_H_
+#define __FREERTOS_PLATFORM_CONFIG_H_
+
+/* Clock config that is platform dependent */
+
```

Comment:
By standards, shouldn't defines be all caps, in this case, CONFIG_TICK_RATE_HZ?

Reply:
This is a FreeRTOS config, it's not done by us